### ISSUE TITLE
Vivaldi crashes when you do a search in the settings menu

### ISSUE THREAD

**Issue by Hydranet** [flathub/com.vivaldi.Vivaldi#80]
I'm using Silverblue and when I do a search in in the settings menu Vivaldi crashes. I tried it with different themes, default themes that come with the browser and ones that don't, both animated and non-animated. I tried the same with the Flatpak version of Vivaldi on a Linux Mint vm I have and there this doesn't happen. 

I tried running Vivaldi from the command-line and then doing a search through the settings menu, , then I see the following two things mentioned.
```
(vivaldi:2): GLib-GIO-ERROR **: 20:12:15.381: Settings schema 'org.gnome.shell' is not installed
[1004/201215.381640:ERROR:scoped_ptrace_attach.cc(27)] ptrace: Operation not permitted (1)
```

**Comment by Hydranet** [comment]
I also tried it on normal Fedora with rpm version, it doesn't occur then. I also tried the Flatpak version on  normal Fedora and the problem also occurs there with the same output as mentioned above for Silverblue.

**Comment by Hydranet** [comment]
This seems to have been solved with the update to 7.0. Not experience this issue anymore.
